### ISSUE TITLE
Deletion of data on offen.dev is not possible

### ISSUE THREAD

**Issue by luckydonald** [offen/offen#544]
<img width="1103" alt="image" src="https://user-images.githubusercontent.com/2737108/105650126-01f62e80-5eb3-11eb-95de-546e886c08ea.png">

```
An error occurred: null is not an object (evaluating 'n.resolution')
getDefaultStats@https://offen.offen.dev/vault/index-4e527f4741.js:13:62786
https://offen.offen.dev/vault/index-4e527f4741.js:13:94653
https://offen.offen.dev/vault/vendor-35fb249183.js:13:18998
Le@https://offen.offen.dev/vault/vendor-35fb249183.js:13:13450
Ge@https://offen.offen.dev/vault/vendor-35fb249183.js:13:13937
https://offen.offen.dev/vault/vendor-35fb249183.js:13:14301
```

# Steps to reproduce

1. Open https://offen.offen.dev/auditorium/
2. Click "Delete"

<img width="223" alt="image" src="https://user-images.githubusercontent.com/2737108/105650172-31a53680-5eb3-11eb-85a2-9054787a6b3a.png">

**Comment by luckydonald** [comment]
I have UBlock Origin installed but as the analytics was tracked I don't think that's the reason.
I also checked the opt-out button, and that one works, and seem to delete the data just fine. Opt in again works too, but delete still fails.

**Comment by m90** [comment]
Thank you for reporting this issue. I can repro and it seems this got introduced in #539 

Good news is it didn't make into a release yet (we run `offen.offen.dev` off the `development` branch more or less) and it also only caused the UI to break when data was still being deleted.

I'll deploy the fix once 2ec2d50 has been built.
